src/UI/Component/Popover/Popover.php

```diff
@@ -98,4 +98,18 @@ public function getShowSignal();
 	 * @return ReplaceContentSignal
 	 */
 	public function getReplaceContentSignal();
+
+
+	/**
+	 * Get a popover which can be used in fixed places such as the main menu
```

Comment:
Maybe add some further line such as, "This popover will stay fixed when scrolling and therefore remain on the screen".

Introduce an according rule such as: "Popovers with fixed Position MUST only be attached to triggerers with fixed position".